src/m365/teams/commands/team/team-add.spec.ts

```diff
@@ -590,4 +672,121 @@ describe(commands.TEAM_ADD, () => {
+    const userId = 'd0fe0abd-bfe8-4a7d-9957-e8fb2d739f61';
+
+    sinonUtil.restore(accessToken.isAppOnlyAccessToken);
+    sinon.stub(accessToken, 'isAppOnlyAccessToken').returns(true);
+
+    const requestStub = sinon.stub(request, 'post').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/teams`) {
+        return { statusCode: 202, headers: { location: "/teams('79afc64f-c76b-4edc-87f3-a47a1264695a')/operations('8ad1effa-7ed1-4d03-bd60-fe177d8d56f1')" } };
+      }
+      throw 'Invalid request';
+    });
+
+    const getRequestStub = sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/teams('79afc64f-c76b-4edc-87f3-a47a1264695a')/operations('8ad1effa-7ed1-4d03-bd60-fe177d8d56f1')`) {
+        return {
+          "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#teams('79afc64f-c76b-4edc-87f3-a47a1264695a')/operations/$entity",
+          "id": "8ad1effa-7ed1-4d03-bd60-fe177d8d56f1",
+          "operationType": "createTeam",
+          "createdDateTime": "2020-06-15T22:28:16.3007846Z",
+          "status": "succeeded",
+          "lastActionDateTime": "2020-06-15T22:28:16.3007846Z",
+          "attemptsCount": 1,
+          "targetResourceId": "79afc64f-c76b-4edc-87f3-a47a1264695a",
+          "targetResourceLocation": "/teams('79afc64f-c76b-4edc-87f3-a47a1264695a')",
+          "Value": "{\"apps\":[{\"Index\":1,\"Status\":\"InProgress\",\"UpdateTimestamp\":\"2020-06-15T22:28:16.8753199+00:00\",\"Reference\":\"com.microsoft.teamspace.tab.vsts\"},{\"Index\":2,\"Status\":\"InProgress\",\"UpdateTimestamp\":\"2020-06-15T22:28:16.8753199+00:00\",\"Reference\":\"1542629c-01b3-4a6d-8f76-1938b779e48d\"}],\"channels\":[{\"tabs\":[],\"Index\":1,\"Status\":\"NotStarted\",\"UpdateTimestamp\":\"2020-06-15T22:28:14.0279825+00:00\",\"Reference\":\"Class Announcements\"},{\"tabs\":[],\"Index\":2,\"Status\":\"NotStarted\",\"UpdateTimestamp\":\"2020-06-15T22:28:14.0279825+00:00\",\"Reference\":\"Homework\"}],\"WorkflowId\":\"northeurope.695866c1-c68a-435c-b707-432984ec721c\"}",
+          "error": null
+        };
+      }
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, {
+      options: {
+        verbose: true,
+        name: 'Architecture',
+        description: 'Architecture Discussion',
+        ownerIds: userId
+      }
+    });
+    assert.deepEqual(requestStub.getCall(0).args[0].data, {
+      "template@odata.bind": "https://graph.microsoft.com/v1.0/teamsTemplates('standard')",
+      displayName: 'Architecture',
+      description: 'Architecture Discussion',
+      members: [{
+        '@odata.type': '#microsoft.graph.aadUserConversationMember',
+        roles: ['owner'],
+        'user@odata.bind': `https://graph.microsoft.com/v1.0/users('${userId}')`
+      }]
+    });
+    assert(getRequestStub.called);
```

Comment:
Let's do only one assert per test. Otherwise, when the test fails, it's hard to understand which assert fails, unless we specify a custom message on each one.